.lintstagedrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  '!(style|styles|test).js': filenames => [
+    `prettier --write ${filenames.join(' ')}`,
+    `eslint --fix ${filenames.join(' ')}`,
+    `git add ${filenames.join(' ')}`,
```

Comment:
If we want to replicate `PocketDerm`'s config, we can bump `lint-staged` to ^10 and remove the `git add` steps (which are done by default in v10).

Reply:
yep, thanks it is much cleaner now. Note that unlike the version used in PocketDerm I had to write the TSC command as a function, like indicated in the Docs https://www.npmjs.com/package/lint-staged

with simple string it doesnt work. Maybe it was a change for  `10.1.3` version.

Anyway tested every type of file leaving errors and spaces  and works fine